configdialog.cpp

```diff
@@ -518,6 +532,23 @@ void PaletteCustomizeDialog::get_and_save_color(int palette_index) {
 	}
 }
 
+std::wstring PaletteCustomizeDialog::export_palettes() {
+	auto all_palettes = m_palette_repo.get_all_custom_palettes();
+	std::wstring palettes_string = L"";
+
+	for (const auto &palette : all_palettes) {
+		auto name = palette.name;
+		auto *data = palette.data;
+
+		palettes_string.append(palette.name + L": ");
+		palettes_string.append(m_palette_repo.serialize(*data));
+		palettes_string.append(L"\r\n");
+	}
+	palettes_string.append(L"~");
```

Comment:
This doesn't look necessary to me, what's the motivation?

Reply:
Felt like it. (Might use it as a check to see if the end has been reached? Idk there's probably better ways of doing that.)